receiver/sqlqueryreceiver/README.md

```diff
@@ -1,7 +1,7 @@
 # SQL Query Receiver (Alpha)
 
 | Status                   |           |
-|--------------------------|-----------|
+| ------------------------ | --------- |
 | Stability                | [alpha]   |
 | Supported pipeline types | metrics   |
```

Comment:
```suggestion
| Supported pipeline types | logs, metrics   |
```
Would it be accurate to say you'd want to have this receiver support log pipelines?

Reply:
Yes, of course, thanks Antoine :slightly_smiling_face: